File: ScaleformUI_Csharp/PauseMenu/Tabs/SubmenuTab.cs
using System.Collections.Generic;
using System.Drawing;
using CitizenFX.Core.Native;
using CitizenFX.Core;

namespace ScaleformUI.PauseMenu
{
    public class SubmenuTab : BaseTab
    {
        private bool _focused;
        public SubmenuTab(string name) : base(name)
        {
            _type = 1;
        }

        public List<BasicTabItem> Items = new List<BasicTabItem>();
        public int Index { get; set; } = 0;
        public bool IsInList { get; set; }

        public override bool Focused
        {
            get { return _focused; }
            set
            {
                _focused = value;
                //if (!value) Items[Index].Focused = false;
            }
        }

        public void AddLeftItem(TabLeftItem item)
        {
            item.Parent = this;
            LeftItemList.Add(item);
        }
    }
}

File: ScaleformUI_Lua/src/PauseMenu/Tabs/SubmenuTab.lua
SubmenuTab = setmetatable({}, SubmenuTab)
SubmenuTab.__index = SubmenuTab
SubmenuTab.__call = function()
    return "BaseTab", "SubmenuTab"
end

function SubmenuTab.New(name)
    local data = {
        Base = BaseTab.New(name or "", 1),
        Label = name or "",
        TextTitle = "",
        LeftItemList = {},
        Index = 0,
        Focused = false,
        Parent = nil,
    }
    return setmetatable(data, SubmenuTab)
end

function SubmenuTab:AddLeftItem(item)
    item.Parent = self
    self.LeftItemList[#self.LeftItemList + 1] = item
end